internal/view/viewer.go

```
@@ -1499,6 +1932,51 @@ func (v *Viewer) rowBaseStyle(header bool) tcell.Style {
+	}
+	return truncateToWidth(v.cfg.Chrome.PinnedGlyph, 1)
+}
+
+func (v *Viewer) pinnedColumnGlyph() string {
+	if glyph := truncateToWidth(v.cfg.Chrome.PinnedColumnGlyph, 1); glyph != "" {
+		return glyph
+	}
+	return truncateToWidth(v.cfg.Chrome.PinnedGlyph, 1)
+}
+
+func (v *Viewer) layoutRowPinned(rowIndex int) bool {
```

golangci-lint (2.11.4)

[error] func (*Viewer).layoutRowPinned is unused

(unused)